connector_test.go

```diff
@@ -92,34 +92,53 @@ var _ = Describe("Connector", func() {
 				objType := "networkview"
 				ref := ""
 				returnFields := []string{}
-				forcedProxy := false
+				queryParams := QueryParams{forceProxy: false}
 				expectedURLStr := fmt.Sprintf("https://%s:%s/wapi/v%s/%s",
 					host, port, version, objType)
-				urlStr := wrb.BuildUrl(CREATE, objType, ref, returnFields, forcedProxy)
+			FORCED_PROXY:
+				urlStr := wrb.BuildUrl(CREATE, objType, ref, returnFields, queryParams)
 				Expect(urlStr).To(Equal(expectedURLStr))
+				if queryParams.forceProxy == false {
+					queryParams.forceProxy = true // proxy enabled
+					goto FORCED_PROXY
+				}
```

Comment:
I'm afraid I don't see any justifiable reason for using goto here (L98-L104):
- if the test did fail for any reason, you have no way of telling which case (forceProxy true/false) caused the failure.
- there should be two different test cases, one for `forceProxy := true` and one for `forceProxy:=false`, and the two cases should be inside different instances of `It()` blocks.
- even the implementation approach in L108-L126 would have been much more preferable.

Reply:
I have addressed all the comments in PR "modified test cases for force proxy  #72"